.bumpversion.cfg

```diff
@@ -0,0 +1,7 @@
+[bumpversion]
```

Comment:
This needs to be in it's own file and not in setup.cfg because bumpversion clears comments from the entire file when it edits the version